cfme/cloud/stack.py

```diff
@@ -379,7 +394,7 @@ def resetter(self):
 @navigator.register(Stack, 'Details')
 class Details(CFMENavigateStep):
     VIEW = StackDetailsView
-    prerequisite = NavigateToSibling('All')
+    prerequisite = NavigateToAttribute('collection', 'All')
```

Comment:
Yes it works, `collection` is the reference to the [Collection](https://github.com/vatsalparekh/integration_tests/blob/27e4383d04b436057b9f4e8fea635793556d6785/cfme/cloud/stack.py#L298)
 object/class